tensorflow_serving/config/model_server_config.proto

```diff
@@ -29,6 +29,11 @@ message ModelConfig {
 
   // Type of model (e.g. "tensorflow").
   string model_platform = 4;
+
+  // Index of the version policy for the model indicating how many versions of the model to be served at the same time.
+  // The list of available indexes for version policy is defined in file_system_storage_path_source.proto.
+  // The default option of 0 is to serve only the latest version of the model.
+  int32 version_policy = 5;
```

Comment:
Instead of int32, please make the type FileSystemStoragePathSourceConfig.VersionPolicy.